netmiko/cloudgenix/cloudgenix_ion.py

```diff
@@ -0,0 +1,83 @@
+import time
+
+from netmiko import log
+from netmiko.cisco_base_connection import CiscoSSHConnection
+from netmiko.py23_compat import string_types
+
+
+class CloudGenixIonSSH(CiscoSSHConnection):
+    def establish_connection(self):
+        super(CloudGenixIonSSH, self).establish_connection(width=100, height=1000)
+
+    def session_preparation(self, *args, **kwargs):
+        """Prepare the session after the connection has been established."""
+        self.ansi_escape_codes = True
+        self._test_channel_read()
+        self.write_channel(self.RETURN)
+        self.set_base_prompt(delay_factor=5)
+
+    def disable_paging(self):
+        """Cloud Genix ION sets terminal height in establish_connection"""
```

Comment:
CloudGenix ION doesn't have a generic way to disable output paging?

Reply:
Not that I am aware of (nothing obvious in docs that I saw or in the CLI when I was poking around), though I could for sure be wrong.

Reply:
CloudGenix engineers stated they don't have this; they are working on it.